crud-rest-service/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "crud-rest-service",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "nest build",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "npm run kill && nest start",
+    "start:dev": "npm run kill && nest start --watch",
+    "start:debug": "npm run kill && nest start --debug --watch",
+    "start:nodemon": "npm run kill && nodemon --config nodemon.json",
+    "start:prod": "npm run kill && node dist/main",
+    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "kill": "kill-port --port 3000"
+  },
+  "dependencies": {
+    "@nestjs/common": "^9.0.0",
+    "@nestjs/core": "^9.0.0",
+    "@nestjs/mapped-types": "*",
+    "@nestjs/platform-express": "^9.0.0",
+    "@types/joi": "^17.2.3",
+    "class-transformer": "^0.5.1",
+    "class-validator": "^0.13.2",
+    "joi": "^17.6.0",
+    "kill-port": "^2.0.1",
+    "reflect-metadata": "^0.1.13",
+    "rimraf": "^3.0.2",
+    "rxjs": "^7.2.0",
+    "uuid": "^8.3.2"
```

Comment:
uuid haven't types. Install https://www.npmjs.com/package/@types/uuid to better working

Reply:
done